modules/afsql/afsql.h

```diff
@@ -28,7 +28,7 @@
 #include "mainloop-worker.h"
 #include "string-list.h"
 
-#include <dbi/dbi.h>
+#include <dbi.h>
 
```

Comment:
This seems to have been the wrong fix. LIBDBI_CFLAGS is set to /usr/include in my case (e.g. it doesn't contain the dbi subdirectory), which comes from the .pc file installed by libdbi.

Our configure script however encodes /usr/include/dbi in case the pc file is not found, which certainly needs this change.

I think we should have changed configure.ac to match the pc file. I'll create a patch within my current branch to do that.